client/src/Component/views/Methods/index.js

```diff
@@ -114,6 +115,28 @@ class Methods extends Component {
     }
   };
 
+  displayScenario = () => {
+    Swal.fire({
+      type: 'info',
+      html:
+        '<h2 style="text-align:center;">>Your Scenario</h2>' +
```

Comment:
extra > after h2 tag that is rendering with heading

Reply:
Done :smiley:

Reply:
@teenie-quaggard btw, `lose their method selections` , they won't be lost, only lose is the refresh now

Reply:
ohhhh, i forgot about your 12 hour pr!!
how could i? 😹

Reply:
:joy: